demos/tests/run_tests.py

```diff
@@ -17,8 +17,11 @@
 """
 Test script for the demos.
 
-For the tests to work, the test data directory must contain a "ILSVRC2012_img_val"
-subdirectory with the ILSVRC2012 dataset.
+For the tests to work, the test data directory must contain:
+* a "ILSVRC2012_img_val" subdirectory with the ILSVRC2012 dataset;
+* a "BraTS" subdirectory with BraTS 2017 dataset in NIFTI format (see http://medicaldecathlon.com);
```

Comment:
You say it's supposed to be BraTS 2017, but then you link to Medical Segmentation Decathlon. Which is it?

Reply:
I can also (or instead) provide link to dataset archive on Google Drive. Will it be enough?

Reply:
As long as the user knows what to download.